InventoryManagementSystem_SQL/Products/IProduct.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace InventoryManagementSystem
+{
+    public interface IProduct
```

Comment:
as we discussed multiple times, the class name should be more descriptive of what it contains, in your case it should be IProductRepository since as we mentioned before if the class/interface for DB layer it renamed as "....Repository"